astropy/units/imperial.py

```diff
@@ -2,8 +2,15 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
 """
-This package defines colloquially used Imperial units.  By default, they
-are not enabled.  To enable them, do::
+This package defines colloquially used Imperial units.  They are
```

Comment:
To follow the same explicitness as above, maybe:

```text
These units are not available in the top-level `astropy.units` namespace. To use these units, you must import the `astropy.units.imperial` module::
```
and update the code example to do `from astropy.units import imperial` ...